### ISSUE TITLE
Panic during generating report 

### ISSUE THREAD

**Issue by msoedov** [IzakMarais/reporter#7]
```
2016/09/26 17:30:18 http: panic serving 127.0.0.1:51769: Latex failed with output:
goroutine 436 [running]:
net/http.(*conn).serve.func1(0xc820100480)
        /usr/local/opt/go/libexec/src/net/http/server.go:1389 +0xc1
panic(0x3d62e0, 0xc820503490)
        /usr/local/opt/go/libexec/src/runtime/panic.go:443 +0x4e9
github.com/izakmarais/reporter.stopIf(0xf00028, 0xc820503490)
        /Users/miasool/gocode/src/github.com/izakmarais/reporter/report.go:143 +0x4b
github.com/izakmarais/reporter.(*Report).runLaTeX(0xc820304500, 0xc820209f00)
        /Users/miasool/gocode/src/github.com/izakmarais/reporter/report.go:133 +0x237
github.com/izakmarais/reporter.(*Report).Generate(0xc820304500, 0xc820304500)
        /Users/miasool/gocode/src/github.com/izakmarais/reporter/report.go:58 +0x10b
main.serveReport(0xf05998, 0xc820017d40, 0xc8201b82a0)
        /Users/miasool/gocode/src/github.com/izakmarais/reporter/cmd/grafana-reporter/main.go:53 +0x343
net/http.HandlerFunc.ServeHTTP(0x534ee0, 0xf05998, 0xc820017d40, 0xc8201b82a0)
        /usr/local/opt/go/libexec/src/net/http/server.go:1618 +0x3a
github.com/gorilla/mux.(*Router).ServeHTTP(0xc820010500, 0xf05998, 0xc820017d40, 0xc8201b82a0)
        /Users/miasool/gocode/src/github.com/gorilla/mux/mux.go:114 +0x2a8
net/http.serverHandler.ServeHTTP(0xc82006c280, 0xf05998, 0xc820017d40, 0xc8201b82a0)
        /usr/local/opt/go/libexec/src/net/http/server.go:2081 +0x19e
net/http.(*conn).serve(0xc820100480)
        /usr/local/opt/go/libexec/src/net/http/server.go:1472 +0xf2e
created by net/http.(*Server).Serve
        /usr/local/opt/go/libexec/src/net/http/server.go:2137 +0x44e

```

**Comment by msoedov** [comment]
That most likely caused by missed installation of pdflatex. But at another hand we can add a descriptive error message about.

**Comment by IzakMarais** [comment]
Fixed by 964dcfeba0c1fec61b75a1a6411e52955f89f056
